src/components/LogsTab/LogsTab.module.css

```diff
@@ -0,0 +1,72 @@
+.tabExpanded {
```

Comment:
Please rename this module as `index.module.css`.

Reply:
I have fixed it.